packages/mask/src/plugins/Trader/SNSAdaptor/trader/TraderDialog.tsx

```diff
@@ -70,47 +78,59 @@ const useStyles = makeStyles()((theme) => ({
 export function TraderDialog() {
```

Comment:
I removed the `TraderProps` state and added `defaultCoins`. cc @zhouhanseng

Reply:
LGTM.